__tests__/articlesRoutes.tests.js

```diff
@@ -47,3 +47,54 @@ describe("GET /api/articles/:article_id", () => {
     });
   });
 });
+
+describe("GET /api/articles", () => {
+  describe("Happy path", () => {
+    it("should return a 200", async () => {
+      await request(app).get("/api/articles").expect(200);
+    });
+    it("should return an array of objects", async () => {
+      const {
+        body: { articles },
+      } = await request(app).get("/api/articles").expect(200);
+      expect(articles).toBeInstanceOf(Array);
+      expect(articles).not.toHaveLength(0);
+    });
+    it("should return a list of objects that match the article schema", async () => {
+      const {
+        body: { articles },
+      } = await request(app).get("/api/articles").expect(200);
+      articles.forEach((article) => {
```

Comment:
I know you've checked the length in a previous assertion, but any time you're iterating in a test I would stack the expect statements to check the length then iterate in the same test block